index.html

```diff
@@ -18,29 +18,189 @@
-                <a href="#" class="connect_anchor button">
-                    <h3>LET’S CONNECT</h3>
+                <a href="#" class="connect_anchor">
+                    LET’S CONNECT
                 </a>
             </div>
-                
+
             <ul class="social_icons content">
-                <li><a href="#" class="social_icon"><img class="icon" src="images/twitter_icon.svg" alt="twitter"></a></li>
-                <li><a href="#" class="social_icon"><img class="icon" src="images/linkedin_icon.svg" alt="linkedin"></a></li>
-                <li><a href="#" class="social_icon"><img class="icon" src="images/medium_icon.svg" alt="medium"></a></li>
-                <li><a href="#" class="social_icon"><img class="icon" src="images/github_icon.svg" alt="github"></a></li>
-                <li><a href="#" class="social_icon"><img class="icon" src="images/hand_icon.svg" alt="hand icon"></a></li>
-                
+                <li><a href="#" class="social_icon"><img class="icon" src="images/twitter_icon.svg" alt="twitter"></a>
+                </li>
+                <li><a href="#" class="social_icon"><img class="icon" src="images/linkedin_icon.svg" alt="linkedin"></a>
+                </li>
+                <li><a href="#" class="social_icon"><img class="icon" src="images/medium_icon.svg" alt="medium"></a>
+                </li>
+                <li><a href="#" class="social_icon"><img class="icon" src="images/github_icon.svg" alt="github"></a>
+                </li>
+                <li><a href="#" class="social_icon"><img class="icon" src="images/hand_icon.svg" alt="hand icon"></a>
+                </li>
+
             </ul>
         </section>
     </header>
+    <main>
+        <section class="main_section">
+            <div>
+                <img class="image_card" src="/images/tonic_snapshot.png" alt="snapshot tonic">
+            </div>
+            <div class="content_card">
+                <h2 class="title_card">Tonic</h2>
+                <div class="subtitle_card">
+                    <h6 class="first_subtitle_card">Canopy</h6>
+                    <img class="point_icon" src="images/point_icon.png" alt="point">
+                    <h6 class="second_subtitle_card">Back End Dev</h6>
+                    <img class="point_icon" src="images/point_icon.png" alt="point">
+                    <h6 class="second_subtitle_card">2015</h6>
+                </div>
+                <p class="paragraph_card">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <div>
+                    <ul class="technologies">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                </div>
+                <button type="button" class="button_card button"><b>See project</b></button>
+            </div>
+        </section>
```

Comment:
- Great Job implementing the project cards 👍🏽 

- I'd like to kindly suggest that you wrap ALL project cards in a `<section>` tag. This section can be likened to be the container that houses all cards. Adding something like this 👉🏽 `<section class="works-container">` would be great.

- Instead of wrapping individual card with `<section class="main_section">`, why don't we replace the section tag on each card with a div. How about we do this instead 👉🏽 `<div class="main_section">`

Section tags are often used to represent a container.

What do you think about this? 🤔  Can we implement these changes everywhere necessary?

Reply:
Thank you so much, I changed the sections for div and I wrapped them all into the section element.